### DISCUSSION TITLE
是否可以支持修改时间序列的编码/压缩类型

### DISCUSSION THREAD

**Discussion by lpf4254302** [apache/iotdb#6323] [Ideas]
功能需求有两点：
1、支持修改时间序列编码/压缩类型命令
2、支持批量修改历史tsfile文件中某些时间序列的编码/压缩类型

可能的使用场景：
为查询效率、降低存储，修改原来的编码方式，例如原来的都用的plain，要修改成更优的编码

这种是否有必要实现

**Comment by jixuan1989** [comment]
Hi， 有必要的。不过目前还没有该功能。 欢迎加入一起做~

yes it is useful if we can support that, welcome to join to discuss how to implement it.
